debezium-connector-mysql/pom.xml

```diff
@@ -685,6 +745,22 @@
           <database.replica.port>${mysql.gtid.replica.port}</database.replica.port>
         </properties>
       </profile>
-
+      <!--  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      Use the alternative Docker image for MySQL.
+      To use, specify "-Dmysql-ssl" or -Pmysql-ssl on the Maven command line.
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+      <profile>
+          <id>mysql-ssl</id>
```

Comment:
So do we still need to set up a run using this profile for CI? I'm confused by the new test passing as part of this PR's build. An overview description of how this all works would be great.

Reply:
@gunnarmorling There are two approaches for this

* add a separate job into CI matrix build
* add a new execution into failsafe plugin configuration - where gitds etc. are tested. That way it will be automatically picked up and tested